common/dynamicconfig/constants.go

```diff
@@ -673,6 +673,41 @@ const (
 	// TransferQueueMaxReaderCount is the max number of readers in one multi-cursor transfer queue
 	TransferQueueMaxReaderCount = "history.transferQueueMaxReaderCount"
 
+	// CallbackTaskBatchSize is batch size for callbackQueueFactory
+	CallbackTaskBatchSize = "history.callbackTaskBatchSize"
+	// CallbackProcessorFailoverMaxPollRPS is max poll rate per second for callbackQueueFactory
+	CallbackProcessorFailoverMaxPollRPS = "history.callbackProcessorFailoverMaxPollRPS"
+	// CallbackProcessorMaxPollRPS is max poll rate per second for callbackQueueFactory
+	CallbackProcessorMaxPollRPS = "history.callbackProcessorMaxPollRPS"
+	// CallbackProcessorMaxPollHostRPS is max poll rate per second for all callbackQueueFactory on a host
+	CallbackProcessorMaxPollHostRPS = "history.callbackProcessorMaxPollHostRPS"
+	// CallbackProcessorSchedulerWorkerCount is the number of workers in the host level task scheduler for callbackQueueFactory
+	CallbackProcessorSchedulerWorkerCount = "history.callbackProcessorSchedulerWorkerCount"
+	// CallbackProcessorSchedulerActiveRoundRobinWeights is the priority round robin weights used by callback task scheduler for active namespaces
+	CallbackProcessorSchedulerActiveRoundRobinWeights = "history.callbackProcessorSchedulerActiveRoundRobinWeights"
+	// CallbackProcessorSchedulerStandbyRoundRobinWeights is the priority round robin weights used by callback task scheduler for standby namespaces
+	CallbackProcessorSchedulerStandbyRoundRobinWeights = "history.callbackProcessorSchedulerStandbyRoundRobinWeights"
+	// CallbackProcessorUpdateShardTaskCount is update shard count for callbackQueueFactory
+	CallbackProcessorUpdateShardTaskCount = "history.callbackProcessorUpdateShardTaskCount"
```

Comment:
`UpdateShardTaskCount` is a verb group, a better name would be `CountForUpdateShardTask`

Reply:
Thanks, I'll remove.